omod/src/main/webapp/pages/allergy.gsp

```diff
@@ -101,10 +101,22 @@ ${ ui.includeFragment("allergyui", "removeAllergyDialog") }
             <ul>
             <% reactionConcepts.each { reaction -> %>
                 <li>
-                    <input ng-model="reaction${reaction.id}" type="checkbox" id="reaction-${reaction.id}" class="allergy-reaction" name="allergyReactionConcepts" value="${reaction.id}" ng-init="reaction${reaction.id} = ${ allergyReactionConcepts.contains(reaction) }" />
```

Comment:
Was this line removed intensionally!

Reply:
Yes it was so that the logical code expressed can work accordingly